google-cloud-apis/java-apis/maven-apis/resources/META-INF/cloud-apis-java-maven.xml

```diff
@@ -15,4 +15,33 @@
   -->
 
 <idea-plugin>
+    <depends>org.jetbrains.idea.maven</depends>
+
+    <project-components>
+        <component>
+            <implementation-class>com.google.cloud.tools.intellij.cloudapis.maven.CloudLibraryProjectState
```

Comment:
Should we rename this to `CloudLibraryMavenProjectState`?

Reply:
sounds good.